tests/msg_avail/main.c

```diff
@@ -34,28 +36,27 @@ int main(void)
 
     msg_init_queue(msg_queue, MSG_QUEUE_LENGTH);
 
+    puts("[START]");
```

Comment:
I thought about making an issue (or mail on developer) regarding the general _structure_ of tests, but haven't done so for 😞 . My idea (RFC) would be to make tests more consistent in the overall workflow, that is:

- print/puts `[START]` when a tests begins and `[SUCCESS]`or `[FAILURE]` on completion. This should be always be output to allow for our test automation with `pexpect`, hence the `puts()`.
- any other output, which is only useful when running manually, should use `LOG_INFO` or `LOG_DEBUG`. So it can be omitted (or saved) especially for very constraint platforms, such as _arduino duemilanove` with only 2K RAM.
- every tests should also provide a (short) README on what it does, expected outcome. That way no test in-code description that blows up required RAM/flash sizes.
- and more ... 

I'll try to write this down soonish, what do you think?